aws/internal/service/ec2/errors.go

```diff
@@ -55,6 +60,7 @@ const (
 )
 
 const (
+	ErrCodeInvalidSubnetIdNotFound = "InvalidSubnetId.NotFound"
```

Comment:
duplicate?

Reply:
No, this (and _Invalid route table_ and probably others) and the uppercase version are returned from different API calls 😄.

Reply:
😞